src/ott/initializers/nn/initializers.py

```diff
@@ -135,8 +135,11 @@ def update(
     """
     return self.update_impl(state, a, b)
 
-  def init_dual_a(  # noqa: D102
-      self, ot_prob: 'linear_problem.LinearProblem', lse_mode: bool
+  def init_dual_a(
+      self,
+      ot_prob: 'linear_problem.LinearProblem',
```

Comment:
Please remove the `'`

Reply:
I assume there was a reason this was there before. 

The only reason I changed the linear/initializers was because `linear_problem` was imported anyway rather than optionally via type checking as detailed below.

In the `nn/initializers` there is optional type checking
```
if TYPE_CHECKING:
  from ott.problems.linear import linear_problem
```

Reply:
> I assume there was a reason this was there before.

Yes, because of circular imports, but seems it was not necessary; the `'` in `'linear_problem.LinearProblem'` can be therefore removed.

Reply:
Ah sorry, my mistake, this is in `nn`, please keep.